src/transformStateWithClones.js

```diff
@@ -7,7 +7,42 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const history = [];
+  let clone = Object.assign({}, state);
```

Comment:
[NAMING] - use proper names for object copy. Instead of `clone`, a more descriptive name like `stateCopy` would be clearer.